### DISCUSSION TITLE
Disable "slowing down for traffic"?

### DISCUSSION THREAD

**Discussion by crcarmichael** [Courseplay/courseplay#7595] [Q&A] [answered]
Is it possible to disable the "slowing down for traffic" function?  It causes problems when unloading harvesters on the move... the harvester is constantly changing speeds while the unloader is alongside.  I looked through all of the options I could find in the UI and didn't see a way to turn it off. Is there a flag in a config file somewhere I could change?

**Comment by Tensuko** [comment, marked as answer]
Give more offset or use a smaler Header for the Combine if already on the edge of the trailer.
